tests/e2e/forms/repeat-form.wdio-spec.js

```diff
@@ -7,16 +7,29 @@ const loginPage = require('../../page-objects/login/login.wdio.page');
 const commonPage = require('../../page-objects/common/common.wdio.page');
 const reportsPage = require('../../page-objects/reports/reports.wdio.page');
 
-const xml = fs.readFileSync(`${__dirname}/../../forms/repeat-translation.xml`, 'utf8');
-const formDocument = {
-  _id: 'form:repeat-translation',
-  internalId: 'repeat-translation',
-  title: 'Repeat',
+const countForm = fs.readFileSync(`${__dirname}/../../forms/repeat-translation-count.xml`, 'utf8');
+const countFormDocument = {
+  _id: 'form:repeat-translation-count',
+  internalId: 'repeat-translation-count',
+  title: 'Repeat count',
   type: 'form',
   _attachments: {
     xml: {
       content_type: 'application/octet-stream',
-      data: Buffer.from(xml).toString('base64')
+      data: Buffer.from(countForm).toString('base64')
+    }
+  }
+};
+const buttonForm = fs.readFileSync(`${__dirname}/../../forms/repeat-translation-button.xml`, 'utf8');
+const buttonFormDocument = {
+  _id: 'form:repeat-translation-button',
+  internalId: 'repeat-translation-button',
+  title: 'Repeat button',
+  type: 'form',
+  _attachments: {
+    xml: {
+      content_type: 'application/octet-stream',
+      data: Buffer.from(buttonForm).toString('base64')
     }
   }
 };
```

Comment:
This is mostly a repeat of the lines above - let's make a function that takes the filename, id, and title (or make the title the same as the id?) to save the duplication.